client/src/Components/Pages/Auth/Auth.js

```diff
@@ -1,13 +1,11 @@
 import React, { useState} from 'react';
 
 // Components
-import { Login } from './Login';
-import { Register } from './Register';
+import Login from './Login';
+import Register from './Register';
 
 export const Auth = (props) => {
   const [page, setPage] = useState('login')
```

Comment:
I'm not clear on the naming here. What does this do?

Reply:
`Login` and `Register` are the two different forms, conditionally rendered on `Auth`. Will rename to `LoginForm` and `RegistrationForm` and place these in the `Elements` folder